build-tools/libzip-windows/libzip-windows.mdproj

```diff
@@ -26,12 +26,21 @@
   <Import Project="libzip-windows.targets" />
   <Import Project="..\libzip\libzip.targets" />
   <ItemGroup>
-    <ProjectReference Include="..\android-toolchain\android-toolchain.mdproj">
+    <ProjectReference Include="..\android-toolchain\android-toolchain.mdproj" Condition=" '$(HostOS)' != 'Windows' ">
       <Project>{8FF78EB6-6FC8-46A7-8A15-EBBA9045C5FA}</Project>
       <Name>android-toolchain</Name>
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
-  <Target Name="Build" DependsOnTargets="$(BuildDependsOnLocal)" />
+  <Target Name="Build" DependsOnTargets="$(BuildDependsOnLocal)" Condition=" '$(HostOS)' != 'Windows' " />
+  <Target Name="Build"
+          Condition=" '$(HostOS)' == 'Windows' "
```

Comment:
These lines look like they're indented too much; it should be 4 spaces from the `<Target`/6 spaces from the leftmost column.